src/dotnet/commands/dotnet-run/LocalizableStrings.resx

```diff
@@ -138,6 +138,9 @@
   <data name="CommandOptionNoLaunchProfileDescription" xml:space="preserve">
     <value>Do not attempt to use launchSettings.json to configure the application.</value>
   </data>
+  <data name="CommandOptionCLRDescription" xml:space="preserve">
+    <value>Specifies the runtime used to execute the .NET app.</value>
```

Comment:
With changes that went into `release/2.1.4xx`, the help strings are of a particular format to make them consistent.

This should probably be something along the lines of `The execution runtime to use to run the .NET application.`